presto-docs/src/main/sphinx/develop/worker-protocol.rst

```diff
@@ -112,6 +112,17 @@ output buffer zero.
 .. image:: worker-protocol-results.png
   :width: 600
 
+Presto Native Sidecar Endpoints
```

Comment:
I'm not sure this should be added here, as technically this does not belong on the worker, but the sidecar.

The specifics of the HTTP endpoints should be documented as OpenAPI, as there are other benefits (like client stub generations, easy mock generation, and generation of documentation, and schematization that keeps the documentation in sync with the code).  I think we could add a new section which describes the high level details of the Presto sidecar, and links to the OpenAPI documentation for details.

We were planning to add the OpenAPI documentation along with the Java code (reason being, it's integrated to the build through Maven).  My personal opinion is we can wait to add it until we add the Java documentation.  

If we're concerned about merging this code before documentation has been added, then we could leave it here for now, but I'd like to eventually remove it, and we'll likely remove it shortly.

Reply:
> If we're concerned about merging this code before documentation has been added, then we could leave it here for now

It would be nice to add documentation with the code. it is ok if it needs to be moved / reformatted later. That's easier than putting it together from scratch. The way you describe OpenAPI docs sounds super cool. Let's move to that, but in the meantime let's continue writing documentation as we land changes. BTW, can we increase the cadence of updating the docs website?

Reply:
The effect I'm looking for is to have access to the very latest documentation (trunk). I not that, at least something that's no more than a few days old.

Reply:
The OpenAPI documentation will be added in another PR, reverted docs from this PR.